packages/plugins/test/mocha/cli/deploy.spec.ts

```diff
@@ -642,7 +636,7 @@ describe('Deployment Cases', () => {
       deploymentConfig,
       getEmptyDeploymentArtifacts(),
       artifacts,
-      ExecutionMode.LocalNetworkCLI,
+      ExecutionMode.LiveNetworkCLI,
```

Comment:
Changed this b/c our default SphinxContext mocks `isLiveNetwork` to return `true` by default. (This a choice made out of convenience, since `validateProposalNetworks` checks that the networks are live networks)